flink-yarn/src/main/java/org/apache/flink/yarn/YarnResourceManager.java

```diff
@@ -469,6 +460,46 @@ public void onError(Throwable error) {
 		onFatalError(error);
 	}
 
+	// ------------------------------------------------------------------------
+	//  NMClientAsync CallbackHandler methods
+	// ------------------------------------------------------------------------
+	@Override
+	public void onContainerStarted(ContainerId containerId, Map<String, ByteBuffer> map) {
+		log.debug("Succeed to call YARN Node Manager to start container", containerId);
```

Comment:
`log.debug("Succeeded to call YARN Node Manager to start container with id {}.", containerId);`